docs/source/index.rst

```diff
@@ -32,27 +32,15 @@ such topics as:
 * Reading and writing file formats (like CSV, Apache ORC, and Apache Parquet)
 * In-memory analytics and query processing
 
-.. _toc.columnar:
-
-.. toctree::
-   :maxdepth: 2
-   :caption: Specifications and Protocols
-
-   format/Versioning
-   format/Columnar
-   format/Flight
-   format/Integration
-   format/CDataInterface
-   format/CStreamInterface
-   format/Other
+**To learn how to use Arrow refer to the documentation specific to your
+target environment.**
 
 .. _toc.usage:
 
 .. toctree::
-   :maxdepth: 2
-   :caption: Libraries
+   :maxdepth: 1
+   :caption: Supported Environments
```

Comment:
I don't think that "Libraries" is the best word from a user point of view. I can see how they are different libraries implementing arrow, but from the user point of view "Arrow" is probably seen as on thing that supports multiple different target languages/platforms/environments.

I think that "Supported Platforms" was probably the one that involved the less cognitive load for a reader as that's usually the title you look for when interested in known if a software works on your system, but in the end I went for "Supported Environments" as Platforms seemed to convey too much an OS/Hardware meaning.

Reply:
@nealrichardson Thoughts?